core/src/main/java/cz/xtf/core/bm/BinaryBuild.java

```diff
@@ -81,26 +53,6 @@ public BinaryBuild(String builderImage, Path path, Map<String, String> envProper
 		this.bc = this.createBcDefinition();
 	}
 
-	@Override
-	public void build(OpenShift openShift) {
```

Comment:
this breaks API, no? It should be at least replaced with abstract method

Reply:
Yeah good catch. If an object typed as `BinaryBuld` instead of `ManagedBuild` then yes.